File: SleepDebtCalc.py
#Slightly more familiar with python. 
#Instead of writing if/else statement like SleepDebtCalc.js, create dictionary{day:ideal hours of sleep}
day_sleep_ideal = {'Monday':8, 
                   'Tuesday':8,
                   'Wednesday':8,
                   'Thursday':8,
                   'Friday':8,
                   'Saturday':10,
                   'Sunday':10}

#Actual sleep hours dictionary
day_sleep_actual = {
    'Monday':3,
    'Tuesday':5,
    'Wednesday':4,
    'Thursday':8,
    'Friday':6,
    'Saturday':10,
    'Sunday':9}


def sleep_debt(day):
    if day_sleep_actual[day] == day_sleep_ideal[day]:
        return 'You got the perfect amount of sleep'
    elif day_sleep_actual[day] > day_sleep_ideal[day]:
        return 'You got more sleep than needed'
    else:
        return f'You need to get {day_sleep_ideal[day] - day_sleep_actual[day]} more hours of sleep'
            
print(sleep_debt('Tuesday'))

File: SleepDebtCalc.js
function sleepHours(day){
    if (day === 'monday'){
        return 3
    }else if(day === 'tuesday'){
        return 5
    }else if(day === 'wednesday'){
      return 4
    }else if(day === 'thursday'){
      return 8
    }else if(day === 'friday'){
      return 6
    }else if(day==='saturday'){
      return 10
    }else if(day==='sunday'){
        return 9
    }
}

// Need to fix getActualSleepHours function
function getActualSleepHours(){
    return sleepHours('monday')
    return sleepHours('tuesday')
    return sleepHours('wednesday')
    return sleepHours('thursday')
    return sleepHours('friday')
    return sleepHours('saturday')
    return sleepHours('sunday')
}
console.log(getActualSleepHours())

function getIdealSleepHours(){
    const idealHours = 8
    return idealHours * 7
}

function calculateSleepDebt(){
    let actualSleepHours = getActualSleepHours()
    let idealSleepHours = getIdealSleepHours()
    if (actualSleepHours === idealSleepHours){
        console.log('You got enough sleep tonight!')
    }else if(actualSleepHours > idealSleepHours){
        console.log('You overslept!')
    }else if(actualSleepHours < idealSleepHours){
        console.log(`You need to get more sleep. You owe ${idealSleepHours - actualSleepHours} hours of sleep!`)
    }
}

console.log(calculateSleepDebt())